### PR TITLE
Fix type of environment variables

### PR DESCRIPTION
**Pull request by lucas-p-bernardes**
The environmental variable was been recognized by different types

### CODE REVIEW

File: tflite/tflite-rtsp/demos/object-detection/object-detection.py [hunk 1]
```diff
@@ -14,11 +14,17 @@
 
 
 ## Read Environment Variables
-USE_HW_ACCELERATED_INFERENCE = os.environ.get("USE_HW_ACCELERATED_INFERENCE")
+USE_HW_ACCELERATED_INFERENCE =True
+
+## The system return the variable as String, so it's necessary
```
Comment (brunoaamello) [inline tflite/tflite-rtsp/demos/object-detection/object-detection.py]:
```suggestion
## The system returns the variables as Strings, so it's necessary to convert them where we need the numeric value
```

**Review by brunoaamello — CHANGES_REQUESTED** [review]
Hello @lucasbernardestoradex,

Thanks for these corrections.

My comments are just related to wording.

Please also amend the current commit so it refers to the environment variables as "environment variables" and not "environmental variables"

**Comment by lucas-p-bernardes** [comment]
Hello @leonheldattoradex, could you review and approve these merge request? I'm not allowed to do that.

**Review by brunoaamello — APPROVED** [review]
Thanks @lucasbernardestoradex.

This MR looks good to me.

**Comment by lucas-p-bernardes** [comment]
@leonheldattoradex Can you please merge this?

File: tflite/tflite-rtsp/demos/object-detection/object-detection.py [hunk 2]
```diff
@@ -14,11 +14,17 @@
 
 
 ## Read Environment Variables
-USE_HW_ACCELERATED_INFERENCE = os.environ.get("USE_HW_ACCELERATED_INFERENCE")
+USE_HW_ACCELERATED_INFERENCE =True
```
Comment (leonheldattoradex) [inline tflite/tflite-rtsp/demos/object-detection/object-detection.py]: Space before `True`

File: tflite/tflite-rtsp/demos/object-detection/object-detection.py [hunk 3]
```diff
@@ -14,11 +14,17 @@
...
+
+## The system returns the variables as Strings, so it's necessary to convert them where we need the numeric value
+if os.environ.get("USE_HW_ACCELERATED_INFERENCE") == "0":
+    USE_HW_ACCELERATED_INFERENCE = False
 
 MINIMUM_SCORE = os.environ.get("MINIMUM_SCORE")
 if not MINIMUM_SCORE:
     MINIMUM_SCORE = 0.55
+else:
```
Comment (leonheldattoradex) [inline tflite/tflite-rtsp/demos/object-detection/object-detection.py]: This cam be simplified to `MINIMUM_SCORE = float(os.environ.get("MINIMUM_SCORE", default=0.55))`.

Comment (leonheldattoradex) [reply tflite/tflite-rtsp/demos/object-detection/object-detection.py]: Same in line 41.

**Review by leonheldattoradex — COMMENTED** [review]
Please change `environmental` to `environment`.

**Comment by lucas-p-bernardes** [comment]
@leonheldattoradex I fixed the name "environmental" and the definition of the environment variables.

**Comment by leonheldattoradex** [comment]
@lucasbernardestoradex this is the best commit I've ever got from an intern. Not only you applied my suggestion, you did for *every* other case where it applied. Today is a truly glorious day.

**Pull request merged**